.github/workflows/main.yaml

```diff
@@ -364,6 +382,87 @@ jobs:
           kubectl wait --for=delete deployment/test-chart --timeout=2m
           kubectl wait --for=delete deployment/replicated --timeout=2m
 
+      # validate status informers
+      - name: Create empty status informers for validation
+        run: |
+          cat << EOF > test-values.yaml
+          replicated:
+            statusInformers: []
+          EOF
+    
+      - name: Install via Helm as subchart in production mode and pass empty status informers
+        run: |
+          helm install test-chart oci://registry.replicated.com/$APP_SLUG/$CHANNEL_SLUG/test-chart --set replicated.integration.enabled=false -f test-values.yaml --wait --timeout 2m
+
+          COUNTER=1
+          while ! kubectl logs deploy/replicated | grep -q 'received an inform event:.*\[\]types\.StatusInformerString{}'; do
+            ((COUNTER += 1))
+            if [ $COUNTER -gt 60 ]; then
+              echo "Did not receive empty status informers"
+              kubectl logs deploy/replicated
+              exit 1
+            fi
+            sleep 1
+          done
+
+      - name: Upgrade via Helm as subchart in production mode to use default status informers
+        run: |
+          helm upgrade test-chart oci://registry.replicated.com/$APP_SLUG/$CHANNEL_SLUG/test-chart --set replicated.integration.enabled=false --wait --timeout 2m
+
+          COUNTER=1
+          while ! kubectl logs deploy/replicated | grep -qv 'received an inform event:.*\[\]types\.StatusInformerString{}'; do
```

Comment:
how is this different than empty status informers?

Reply:
shouldn't there be informers that are generated automatically here?

Reply:
this is inverting the match and validating that it's _not_ empty.
```
     -v, --invert-match
             Selected lines are those not matching any of the specified patterns.
```

Reply:
i believe so, yes

Reply:
after some further discussion, we're going to test these cases a different way by logging the generated status informers in `GenerateStatusInformersForManifest` as that information will provide additional value when debugging